plugins/config/repository.yaml

```diff
@@ -24,6 +24,43 @@ repositories:
 # wildcards are allowed in name, entries are processed in top-down order
 # first entry matching the name is used
 projects:
+    # Use nightlies in all main branches
+    - name: ignition-garden
+      repositories:
+          - name: osrf
+            type: stable
+          - name: osrf
+            type: prerelease
+          - name: osrf
+            type: nightly
+    - name: ignition-gazebo7
+      repositories:
+          - name: osrf
+            type: stable
+          - name: osrf
+            type: prerelease
+          - name: osrf
+            type: nightly
```

Comment:
Mixing prerelease and nightly goes :boom: Let's keep just nightly

Reply:
e997bd10b8fe6ba13a024378af045adf44c6eaed